packages/components/src/modal/style.scss

```diff
@@ -135,7 +135,7 @@
 
 	&.hide-header {
 		margin-top: 0;
-		padding-top: $grid-unit-30;
+		padding-top: $grid-unit-50;
```

Comment:
Should this be `$grid-unit-40` to match right, bottom, left?

Reply:
Yeah I guess it should (fixed). Initially I intentionally made it a bit larger because it felt bottom heavy otherwise. But revisiting it today, you're right, and this works too:

<img width="549" alt="Screenshot 2023-05-04 at 13 40 08" src="https://user-images.githubusercontent.com/1204802/236193567-84e0ec4f-d016-495e-a0a2-dcba85266927.png">